media/static-bundles.json

```diff
@@ -1079,7 +1081,8 @@
     {
       "files": [
         "js/base/mozilla-modal.js",
-        "js/firefox/new/scene1.js"
+        "js/firefox/new/scene1.js",
+        "js/firefox/new/scene1-macos-attribution.js"
```

Comment:
Reading the discussion in the bug I understand that the current ask is for this to only apply to `/firefox/new/`, but my gut says this code should really belong in the [stub attribution bundle](https://github.com/mozilla/bedrock/blob/master/media/static-bundles.json#L1146-L1152) along with the windows code? Less because it's neater code-wise, but more because that bundle is isolated to do one thing, so if something throws an error it will not halt excecution of the page JS or break basic functionality.

I'd be interested in @pmac's thoughts here too.